geometry_msgs/Pose face_det
geometry_msgs/Pose leg_det
---
bool human_detected
